search/services/index.py

```diff
@@ -98,8 +98,9 @@ class SearchSession(object):
     # use SSL. Presumably we will use HTTP Auth, or something else.
 
     def __init__(self, host: str, index: str, port: int=9200,
-                 scheme: str='http', **extra) -> None:
-        """
+                 scheme: str='http', user: str=None, password: str=None,
+                 **extra) -> None:
+
         Initialize the connection to Elasticsearch.
```

Comment:
Looks like we're getting a syntax error here?

Reply:
Maybe missing opening quotes: ```"""```

Reply:
Thanks, I accidentally removed those during the merge